examples/test_examples.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Planet Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test example scripts in this directory
+
+This is not an example script. This is a pytest module that provides automated
+testing of the example scripts in this directory to ensure they are always
+up-to-date and run successfully.
+
+YAY automated testing and up-to-date examples!
```

Comment:
Sweet! LGTM, this aligns with our previous discussions on the topic outside of GH